Doc/library/errno.rst

```diff
@@ -659,12 +659,189 @@ defined by the module.  The specific list of defined symbols is available as
 
    Quota exceeded
 
+
 .. data:: EQFULL
 
    Interface output queue is full
 
    .. versionadded:: 3.11
 
+
+.. data:: ENOMEDIUM
+
+   No medium found
+
+   .. versionadded:: 2.7
```

Comment:
If the constant exists since Python 3.0, there is no need to add a "versionadded" markup: you can remove it. In Python 3, we don't mention Python 2.7. Same remark for the other "versionadded:: 2.7".

Reply:
Okay, I've remove them :)